src/bit-systems/audio-debug-system.ts

```diff
@@ -154,6 +154,13 @@ export function audioDebugSystem(world: HubsWorld) {
   navMeshExitQuery(world).forEach(navEid => {
     removeDebugMaterial(world, navEid);
   });
+  navMeshEnterQuery(world).forEach(navEid => {
+    if (isEnabled) {
+      addDebugMaterial(world, navEid);
+    } else {
+      removeDebugMaterial(world, navEid);
+    }
+  });
```

Comment:
Can't this code be simplified and straightforward more?

```typescript
  navMeshEnterQuery(world).forEach(navEid => {
    if (isEnabled) {
      addDebugMaterial(world, navEid);
    } else {
      removeDebugMaterial(world, navEid);
    }
  });
  if (isEnabled && uniforms) {
    navMeshEnterQuery(world).forEach(navEid => {
      isEnabled && addDebugMaterial(world, navEid);
    });
    ...
```

For example, I'm not sure if the second `navMeshEnterQuery(world)` returns non-empty array because the first `navMeshEnterQuery(world)` is always called.